pipeline_dp/dp_computations.py

```diff
@@ -571,3 +571,68 @@ def std(self) -> float:
     @property
     def sensitivity(self) -> float:
         return self._mechanism.l2_sensitivity
+
+
+@dataclass
+class Sensitivities:
+    """Contains sensitivities of the additive DP mechanism."""
+    l0: Optional[int] = None
+    linf: Optional[float] = None
+    l1: Optional[float] = None
+    l2: Optional[float] = None
```

Comment:
Why do we want to have one class that contains all sensitivities at the same time? Wouldn't it be cleaner to have different classes for l1 and l2 sensitivities and include all necessary elements in each of them?